### ISSUE TITLE
Error: Undefined Index

### ISSUE THREAD

**Issue by sarahschuetz** [verbb/formie#844]
### Error:
https://github.com/verbb/formie/blob/7390f6918567060113174eee461af70299ca1805/src/services/Stencils.php#L233
throws `error: Undefined index: submitActionEntry` running `php craft project-config/apply --force`.

**Version: 1.5.15**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by development-lameco** [comment]
When will the update be released officially? Or can it do any harm using the suggested composer.json for future updates?

**Comment by engram-design** [comment]
I'll get a new version out over the weekend, likely one of the last few for Craft 3 (except major bug fixes and security issues). No harm in including the develop version though in the meantime, but eventually you should switch back to a proper version number.

**Comment by engram-design** [comment]
Fixed in [1.5.16](https://github.com/verbb/formie/releases/tag/1.5.16)
